resources/js/Pages/Users.tsx

```diff
@@ -212,61 +278,93 @@ export default function Users({ auth }: PageProps) {
+                            onClick={() => showUserPassword.current?.close()}
+                        >
+                            Close
+                        </button>
+                    </div>
+                </div>
+            </dialog>
+
             <dialog ref={deleteUserModal} className="modal">
                 <div className="modal-box">
                     <h3 className="font-bold text-xl">Delete User?</h3>
                     <p className="py-4"></p>
                     <div className="modal-action">
-                        <button className="btn btn-warning">Delete</button>
+                        <button
+                            className="btn btn-warning"
+                            onClick={deleteUser}
+                        >
+                            Delete
+                        </button>
                         <form method="dialog">
                             {/* if there is a button in form, it will close the modal */}
-                            <button className="btn">Close</button>
+                            <button
+                                className="btn"
+                                onClick={() => setTargetUser(null)}
+                            >
+                                Close
+                            </button>
                         </form>
                     </div>
                 </div>
             </dialog>
 
             {/* Toasts */}
-            <div className="toast transition-opacity duration-500 ease-out opacity-0">
-                <div className="alert alert-success">
-                    <CheckCircleIcon className="h-6" />
-                    <span>User created!</span>
-                </div>
-            </div>
-
-            <div className="toast transition-opacity duration-500 ease-out opacity-0">
-                <div className="alert alert-success">
-                    <CheckCircleIcon className="h-6" />
-                    <span>User updated!</span>
-                </div>
-            </div>
-
-            <div className="toast transition-opacity duration-500 ease-out opacity-0">
-                <div className="alert alert-success">
-                    <CheckCircleIcon className="h-6" />
-                    <span>User password reset!</span>
-                </div>
-            </div>
-
-            <div className="toast transition-opacity duration-500 ease-out opacity-0">
-                <div className="alert alert-success">
-                    <CheckCircleIcon className="h-6" />
-                    <span>User deleted!</span>
+            {toast.isVisible && (
+                <div
+                    className={`toast transition-opacity duration-500 ease-out ${
+                        toast.isVisible ? "opacity-100" : "opacity-0"
+                    }`}
+                >
+                    <div className={`alert alert-${toast.type}`}>
+                        <CheckCircleIcon className="h-6" />
```

Comment:
When its an error, can you render `<ExclamationCircleIcon className="h-6" />` instead?